man/localectl.xml

```diff
@@ -50,10 +50,8 @@
 
     <para>Note that the changes performed using this tool might require the initrd to be rebuilt to take
     effect during early system boot.  The initrd is not rebuilt automatically by
-    <filename>localectl</filename>, this task has to be performed manually, usually using a tool like
-    <citerefentry
-    project='man-pages'><refentrytitle>dracut</refentrytitle><manvolnum>8</manvolnum></citerefentry>.
-    </para>
+    <filename>localectl</filename>, this task has to be performed manually, usually by reinstalling the
+    distribution's kernel package.</para>
```

Comment:
Maybe also say "see the documentation of the specific initrd implementation" here? I'm not sure if reinstalling the kernel package should be recommended for this purpose.

Reply:
Hmm, reinstalling the kernel package seems like the safest recommendation though. Since the exact same hooks will be executed to build and install the initrd. If the initrd is executed directly you might end up with a different result than if you had reinstalled the kernel package.

Reply:
(I'll leave the final decision to others)